src/dataframe/dataframe.jl

```diff
@@ -302,30 +296,28 @@ end
 # df[MultiRowIndex, MultiColumnIndex] => DataFrame
 function Base.getindex(df::DataFrame, row_inds::AbstractVector, col_inds::AbstractVector)
     selected_columns = index(df)[col_inds]
-    new_columns = Any[dv[row_inds] for dv in columns(df)[selected_columns]]
+    new_columns = AbstractVector[dv[row_inds] for dv in columns(df)[selected_columns]]
     return DataFrame(new_columns, Index(_names(df)[selected_columns]))
 end
 
 # df[:, SingleColumnIndex] => AbstractVector
 # df[:, MultiColumnIndex] => DataFrame
 function Base.getindex(df::DataFrame, row_ind::Colon, col_inds)
-    Base.depwarn("indexing with colon as row will create a copy in the future" *
-                 " use df[col_inds] to get the columns without copying", :getindex)
+    Base.depwarn("Sndexing with colon as row will create a copy in the future. " *
```

Comment:
I love Sndexing into data frames

Reply:
This is a new operation I am working on. A combination of "Selecting" and "Indexing" in one go with a bit more of the latter than the former.